code/game/machinery/computer/labor.dm

```diff
@@ -0,0 +1,165 @@
+var/list/labor_console_categories = list(
+	"Command" = command_positions,
+	"Civilian" = civilian_positions,
+	"Security" = security_positions,
+	"Engineering" = engineering_positions,
+	"Medical" = medical_positions,
+	"Science" = science_positions,
+	"Cargo" = cargo_positions,
+	)
+
+/obj/machinery/computer/labor
+	name = "Labor Administration Console"
+	desc = "According to the manual, you need to take a six-week Labor Administration Associate Training Course before you're qualified to navigate this console's complex interface. Being a HoP is hard work."
+	icon = 'icons/obj/computer.dmi'
+	icon_state = "comm_logs"
+	light_color = LIGHT_COLOR_GREEN
+	req_access = list(access_hop)
```

Comment:
Although functionally similar, arguably access_change_ids might be better

Reply:
Unsure, this labor management thing is closer to the HoP's spirit than some hypothetical not-HOP-but-still-can-give-all-access job